augur/data/colors.tsv

```diff
@@ -1,3 +1,4 @@
+region	asia	#447CCD
```

Comment:
I just chose the [color used in measles](https://github.com/nextstrain/measles/blob/957fc744c64b8f5a722b5c525687d0746755add6/phylogenetic/defaults/colors.tsv#L2) so I'm not attached to it in anyway. Feel free to suggest other colors.